app/scripts/modules/core/src/pipeline/config/stages/pipeline/pipelineStage.js

```diff
@@ -18,6 +18,7 @@ module.exports = angular.module('spinnaker.core.pipeline.stage.pipelineStage', [
       controllerAs: 'pipelineStageCtrl',
       templateUrl: require('./pipelineStage.html'),
       executionDetailsUrl: require('./pipelineExecutionDetails.html'),
+      defaultTimeoutMs: 12 * 60 * 60 * 1000, // 12 hours
```

Comment:
I know we have to put this here... but I hate that we have to put this here.

Reply:
I hate so too @jrsquared ... 

Once upon a time we had talked about sourcing some of these details from orca.

Reply:
I was thinking the same thing (pulling from orca)